hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerStateManagerImpl.java

```diff
@@ -415,6 +415,11 @@ public void updateDeleteTransactionId(
           deleteTransactionMap.entrySet()) {
         final ContainerInfo info = containers.getContainerInfo(
             transaction.getKey());
+        if (info == null) {
+          LOG.warn("can not find container id {}, transaction id is {}",
```

Comment:
can not -> Cannot ,   container id -> container

Reply:
thanks @ChenSammi for the review, will fix this comment in a new commit